samples/c2pa/C2paPlayer/C2paTimeline/C2paTimelineFunctions.js

```diff
@@ -84,20 +84,30 @@ let createTimelineSegment = function (
             .getPropertyValue('--c2pa-failed')
             .trim();
     } else {
+        const c2paInvalidButton = document.querySelector('.c2pa-menu-button button')
         if (verificationStatus == 'true') {
```

Comment:
This doesn't seem like part of this PR, but is `verificationStatus` supposed to be a string instead of a boolean?

Reply:
um I'm not sure but this works as expected ,  @spetrangeli ?

Reply:
`verificationStatus` can be `true` (when validation passes), `false` (when it fails), `unkown` (when we can't tell, for example during seeking)

Reply:
Oh ok, I didn't realize it was not binary.